spec/models/user_spec.rb

```diff
@@ -74,4 +76,10 @@
       expect(user).to be_valid
     end
   end
+
+  context 'Scopes' do
+    it '.opt_in' do
+      expect(User.opt_in).to match_array([user, user2])
```

Comment:
A more robust approach would be `expect(User.opt_in).to_not include(opted_out_user)`. Otherwise this will break if someone adds more user factories to the test later.